### ISSUE TITLE
Dashboard Authorize Proxies form field dropdown add label

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#3039]
### Descriptive summary

From the Accessibility 360 Report:

> This form field is not labeled.
> 
> All form fields should have labels (typically via a label tag) which are properly associated with their fields (via the `for` attribute). The best approach is to use this method and delete the "sr-only" class, but it is also acceptable to move the "sr-only" class to the link tag.

![authorize-proxies](https://user-images.githubusercontent.com/3020266/39711026-1d7a4706-51e4-11e8-91dc-9fdc0aa5d03a.png)

### Rationale

Accessibility improvement.

### Tasks
- [x] Form field label added, and/or move the `sr-only` class to the link tag.
